### PR TITLE
fix: fix imageUrl in Image module to generate images from picsum photos #990

### PR DESCRIPTION
**Pull request by bitalec**
Summary of changes:

- Added a new method urlPicsumPhotos() to generate random images from the  https://picsum.photos website.
- Created necessary tests to ensure proper functionality.
- Renamed the imageUrl() method in urlLoremFlickr() to better distinguish between the different websites used for image generation.

### CODE REVIEW

File: include/faker-cxx/image.h [hunk 1]
```diff
@@ -34,14 +34,34 @@ enum class ImageCategory
...
  * @code
- * faker::image::imageUrl() // "https://loremflickr.com/640/480"
- * faker::image::imageUrl(800, 600) // "https://loremflickr.com/800/600"
- * faker::image::imageUrl(800, 600, ImageCategory::Animals) // "https://loremflickr.com/800/600/animals"
+ * faker::image::urlLoremFlickr() // "https://loremflickr.com/640/480"
+ * faker::image::urlLoremFlickr(800, 600) // "https://loremflickr.com/800/600"
+ * faker::image::urlLoremFlickr(800, 600, ImageCategory::Animals) // "https://loremflickr.com/800/600/animals"
  * @endcode
  */
-FAKER_CXX_EXPORT std::string imageUrl(unsigned width = 640, unsigned height = 480,
```
Comment (cieslarmichal) [inline include/faker-cxx/image.h:42]: please leave imageUrl as it was and inside it call either loremFlickr or Picsum function randomly

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
good job, one comment

**Review by cieslarmichal — APPROVED** [review]

**Comment by cieslarmichal** [comment]
Please run clang format

**Comment by bitalec** [comment]
Sorry, i think clang format is run now

**Pull request merged**